### PR TITLE
database change

### PR DESCRIPTION
**Pull request by cleverdumb**
got rid of the need of sqlite, using sqlite3

### CODE REVIEW

**Review by brandonetter — APPROVED** [review]
lgtm 👍🏼

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 0",
-    "start": "node server/server.js"
+    "start": "node server/server.js",
+    "build": "npm i && touch database-temp/develop.db"
```
Comment (Thenlie) [inline package.json:9]: Should we use `npm ci` here instead? https://docs.npmjs.com/cli/v8/commands/npm-ci

Comment (cleverdumb) [reply package.json:9]: ok

Comment (cleverdumb) [reply package.json:9]: how tho should i start a new branch idk how to edit it

**Pull request merged**